configure.ac

```diff
@@ -1353,8 +1368,9 @@ AC_CHECK_HEADER([sys/mman.h],
 AC_CHECK_FUNC([pwrite], [AC_DEFINE([HAS_PWRITE])])
 
 ## -fdebug-prefix-map support by the C compiler
-AS_CASE([$host],
-  [*-*-mingw32|*-pc-windows], [cc_has_debug_prefix_map=false],
+AS_CASE([$CC,$host],
+  [*,*-*-mingw32|*,*-pc-windows], [cc_has_debug_prefix_map=false],
```

Comment:
Please could this be split into two lines - I think it should either be one regexp covering all three platforms (which would be too long a line) or three separate regexps all setting `cc_has_debug_prefix_map=false`, but not a mix of the two!

Reply:
Done.